chesster.js

```diff
@@ -1351,7 +1349,10 @@ controller.on('ambient', function(bot, message) {
             return;
         }
         try{
-            var result = spreadsheets.parse_result(message.text, results_options);
+            var result = 
+                spreadsheets.parse_result(
+                    message.text, 
+                    config.gamelinks[channel.name]);
```

Comment:
You are correct. I should be. This would have led us to parsing the results in the wrong channel. Thanks for pointing it out.